### DISCUSSION TITLE
How do you change colors for XML/HTML tags?

### DISCUSSION THREAD

**Discussion by ahait** [izhangzhihao/intellij-rainbow-brackets#2624] [Q&A] [answered]
it's not possible with current UI.
Editor just selects something standard, even after I changed all the colors for all the other kind of brackets in the settings.

[a comment by an automated account is not shown]

**Comment by izhangzhihao** [comment, marked as answer]
Change the colors by:
<img width="982" alt="image" src="https://user-images.githubusercontent.com/12044174/234530510-e56686a4-655a-455c-8cf0-d1f42fb6d0af.png">


Or if you want a random color pattern:

<img width="987" alt="image" src="https://user-images.githubusercontent.com/12044174/234530629-1d5d418a-1cab-466b-b333-749959faf510.png">

**Comment by ahait** [comment]
Just bought the license for a year! Great product by the way.
These screenshots do not work for XML/HTML brackets -  they only work for "round brackets" 
But I will try again.

**Reply by izhangzhihao** [reply]
Thanks for the support! I will update the description in the next release.

**Reply by izhangzhihao** [reply]
And the two screenshots work for sure, I just tested these two features minutes ago.

**Comment by ahait** [comment]
AHH I see. It's because I did not use "Round Brackets" at all, only square and squigly....  
But it's problematic that I HAVE TO use round brackets in order to set the colors of the XML/HTML brackets - it's a little weird UX
